INSTALLFEST.md

```diff
@@ -559,6 +558,17 @@ sudo apt-get install libffi-dev
     # if you do not get the message 'rbenv 1.1.0' refer to an instructor
     ```
 
+1.  Next, we will install a default gem, `bundler`.
+    ```bash
+    # OSX and LINUX
+    touch ~/.rbenv/default-gems
+    ```
+    then:
+    ```bash
+    # OSX and LINUX
+    echo bundler < ~/.rbenv/default-gems
+    ```
+
```

Comment:
@payne-chris-r It is incorrect.  Please see my comment on the issue I posted to lm01-planning.